_typos.toml

```diff
@@ -0,0 +1,14 @@
+[files]
+extend-exclude = [
+    "*.json",
+    "*.js",
+    "*.ipynb",
+    "src/zenml/zen_stores/migrations/versions/",
+    "tests/unit/materializers/test_built_in_materializer.py",
+    "tests/integration/functional/cli/test_pipeline.py",
+    "src/zenml/zen_server/dashboard/",
+    "examples/llm_finetuning/lit_gpt/"
```

Comment:
These need updating. They relate to the core ZenML package, not this repository.